### ISSUE TITLE
Upgrade quickstarts

### ISSUE THREAD

**Issue by babadofar** [jenkins-x/jx#8423]
The quickstart applications are sample applications that contain all the things that are needed run in Jenkins X. They are used both for demo purposes, and as starting points when creating new applications. Many of the quickstarts were created a long time ago, and needs to be updated to reflect the latest news. 
This goes for both the application frameworks and dependencies, the Dockerfile and perhaps 

Quickstart org: https://github.com/jenkins-x-quickstarts  

Quickstarts will use pipelines from packs folder in pipeline catalog https://github.com/jenkins-x/jx3-pipeline-catalog/tree/master/packs

**Comment by rutviklhase** [comment]
hello! is anyone working on this right now? Would love to take this one up!

**Comment by babadofar** [comment]
I suggest that you start out selecting a quickstart for a language that you are familiar with, one example app that is badly outdated is the one for python, here https://github.com/jenkins-x-quickstarts/python-http   

-  check out the repo and run the application on your own machine. 
-  Check that the app runs on the current version of the language, and that the dependencies are up to date. 
-  try to see if you can find alternative implementations for the code given, that are more relevant, secure, or more popular for other reasons. For instance, check if there are new libraries for handling the  "liveness probe" endpoint, used in Kubernetes to verify that the app is running.   
- Once you have an application that is up to date, secure and easy to handle, you can make a PR on the quickstart repo. 

Next, look at the pack used to import the project into Jenkins X - https://github.com/jenkins-x/jx3-pipeline-catalog/tree/master/packs/python
- check the Dockerfile - can you find a more secure base image? distroless?
- check the helm chart under /charts - is it using any deprecated Kubernetes CRDs

You can make a fork of the pipeline repo, and the quickstart you changed, and use them in your own Jenkins X cluster installation, to test that it works.  A guide on how to do that is found here https://jenkins-x.io/v3/about/extending/#pipeline-catalog

**Comment by iamritikbhardwaj** [comment]
Hello i would like to contribute as well is it possible to assign it to two people i am new to open source so sorry in advance if it's a silly question. also thanks in advance /assign

**Comment by msvticket** [comment]
It's easiest that you just look at the list of repos: https://github.com/orgs/jenkins-x-quickstarts/repositories
GitHubs automatic marking of the main language tend to be correct

**Comment by msvticket** [comment]
It really is up to you. I suppose the top priority would be quickstarts for popular frameworks but where the repo hasn't been updated in a long time. So I guess both of these are good.

**Comment by msvticket** [comment]
No, it's fine to create the PR directly. You can assign it to me
